modules/admin-ui-picard/app/src/components/events/partials/wizards/StartTaskGeneralPage.js

```diff
@@ -0,0 +1,175 @@
+import React, {useEffect, useState} from "react";
+import {useTranslation} from "react-i18next";
+import Notifications from "../../../shared/Notifications";
+import {connect} from "react-redux";
+import cn from 'classnames';
+import {getSelectedRows} from "../../../../selectors/tableSelectors";
+
+/**
+ * This component renders the table overview of selected events in start task bulk action
+ */
+const StartTaskGeneralPage = ({ formik, nextPage, selectedRows }) => {
+    const { t } = useTranslation();
+
+    const [allChecked, setAllChecked] = useState(true);
+    const [selectedEvents, setSelectedEvents] = useState(selectedRows);
+
+    useEffect(() => {
+        // Set field value for formik on mount, because initially all events are selected
+        formik.setFieldValue('events', selectedEvents);
+    }, []);
+
+    // Check if an event is in a state that a task on it can be started
+    const isStartable = event => {
+        return event.event_status.toUpperCase().indexOf('PROCESSED') > -1
+            || event.event_status.toUpperCase().indexOf('PROCESSING_FAILURE') > -1
+            || event.event_status.toUpperCase().indexOf('PROCESSING_CANCELED') > -1 || !event.selected;
+    };
+
+    // Check if multiple event is in a state that a task on it can be started
+    const isTaskStartable = events => {
+        for (let i = 0; i < events.length; i++) {
+            if (!isStartable(events[i])) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    // Select or deselect all rows in table
+    const onChangeAllSelected = e => {
+        const selected = e.target.checked;
+        setAllChecked(selected);
+        let changedSelection = selectedEvents.map(event => {
+            return {
+                ...event,
+                selected: selected
+            }
+        });
+        setSelectedEvents(changedSelection);
+        formik.setFieldValue('events', changedSelection);
+    };
+
+    // Handle change of checkboxes indicating which events to consider further
+    const onChangeSelected = (e, id) => {
+        const selected = e.target.checked;
+        let changedEvents = selectedEvents.map(event => {
+            if (event.id === id) {
+                return {
+                    ...event,
+                    selected: selected
+                }
+            } else {
+                return event
+            }
+        });
+        setSelectedEvents(changedEvents);
+        formik.setFieldValue('events', changedEvents);
+
+        if (!selected) {
```

Comment:
add re-checking of all events checked button, if selection of the event caused all events to be checked again